sys/net/lora/Kconfig

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_LORA
+    bool "Configure LoRa PHY"
+    depends on USEMODULE_LORA
+    help
+        Configure LoRa PHY using Kconfig.
+
+if KCONFIG_USEMODULE_LORA
+
+config LORA_PREAMBLE_LENGTH_DEFAULT
+    int "Preamble length"
+    range 8 65537
+    default 8
+    help
+        Configure preamble used in LoRa frame. Each LoRa frame begins with a
+        preamble. It starts with a series of upchirps to cover the whole
+        frequency band of the particular channel assigned. The last two upchirps
+        encode the sync word. Sync word is used to differntiate between LoRa
+        transmissions that use the same frequency bands. The sync word is
+        followed by two and a quarter downchirps, for a duration of 2.25
+        symbols. The total duration of this preamble can be configured between
+        10.25 and 65,539.25 symbol hence the range 8 to 65537.
+
+config LORA_SYMBOL_TIMEOUT_DEFAULT
+    int "Timeout for symbols in seconds"
```

Comment:
this symbol time out is in number of symbols, not in a time unit

Reply:
Oops yes.